.github/workflows/pythonapp.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           python orm migrate --connection mysql
           make test
   lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     name: Lint
```

Comment:
Whats the point of pegging this to a version instead of always testing against the latest ubuntu version?

Reply:
If we always use the latest we can catch issues as soon as the latest version is updated on that container